wandb/integration/tensorboard/log.py

```diff
@@ -300,6 +301,14 @@ def log(tf_summary_str_or_pb, history=None, step=0, namespace="", **kwargs):
             RATE_LIMIT_SECONDS is None
             or timestamp - STEPS["global"]["last_log"] >= RATE_LIMIT_SECONDS
         ):
-            history.add({}, **kwargs)
+            if history is None:
+                wandb.log({})
+            else:
+                history.add({}, **kwargs)
+
         STEPS["global"]["last_log"] = timestamp
-    history._row_update(log_dict)
+
+    if history is None:
+        wandb.log(log_dict, commit=False)
```

Comment:
Lets make an internal version of run._log() so we can know that this came from a special integration.

Reply:
good idea!